.gitignore

```diff
@@ -64,3 +64,9 @@ package-lock.json
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
 ios/Pods/
+
+#Generated files
+#
+generated/
+intermediates/
+outputs/
```

Comment:
Are these changes related to Reactotron?

Reply:
Outputs should not be included in git repo I did that on my side while setting up the project initially.  Should I remove them? Create a separate PR? or let them be?

Reply:
Do not include anything that is not related to this PR.

Reply:
If you think it is useful for all other contributors create another PR and explain when these folders are generated. Optionally you can add these to your local file and tell `git` to ignore the changes to this file.

Reply:
OK I am making it a new PR.